apps/trading/components/rewards-container/reward-card.tsx

```diff
@@ -991,12 +994,20 @@ export const ActiveRewardCard = ({
       ? transferNode.transfer.kind.endEpoch - currentEpoch
       : undefined;
 
+  const clickable = (element: ReactNode) => {
+    const gameLink = transferNode.transfer.gameId
+      ? Links.COMPETITIONS_GAME(transferNode.transfer.gameId)
+      : undefined;
+    if (gameLink) return <Link to={gameLink}>{element}</Link>;
+    return element;
+  };
+
   if (
     !transferNode.transfer.kind.dispatchStrategy &&
     transferNode.transfer.toAccountType ===
       AccountType.ACCOUNT_TYPE_GLOBAL_REWARD
   ) {
-    return (
+    return clickable(
```

Comment:
This is essentially a component but written as a 'normal' function. Please wrap it with normal component syntax or just Link